### PR TITLE
MRC-1912 CI, travis mods, update cypress

### PR DESCRIPTION
**Pull request by vercasson**
- [x] This change is documented in CHANGELOG.md

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,16 +2,16 @@ module.exports = {
   env: {
     browser: true,
```
Comment (vercasson) [inline .eslintrc.js:3]: prettier taking on eslint file :)

File: .travis.yml [hunk 2]
```diff
@@ -9,11 +9,10 @@ language: node_js
 # Run tests against latest node and current long term support node. Only the LTS
 # job is used for `npm publish`, since we don't want to publish the package
 # twice (that would cause an error).
+os: linux
+dist: jammy
```
Comment (vercasson) [inline .travis.yml:13]: More linux and Ubuntu Jammy (Jammy Jellyfish). Same as MEF

File: cypress/e2e/auto-destroy.cy.ts [hunk 3]
```diff
@@ -5,12 +5,12 @@ describe("Auto-Destroy", () => {
     cy.visit("/cypress/basic");
```
Comment (vercasson) [inline cypress/e2e/auto-destroy.cy.ts:6]: all eslint

File: cypress/e2e/overlays-polyline.cy.ts [hunk 4]
```diff
@@ -4,30 +4,19 @@ describe("Overlays: Polyline", () => {
   it("should render a polyline added thru .update", () => {
     cy.visit("/cypress/basic");
```
Comment (vercasson) [inline cypress/e2e/overlays-polyline.cy.ts:5]: all eslint

File: cypress/e2e/placemark-label-mode.cy.ts [hunk 5]
```diff
@@ -10,23 +10,21 @@ describe("Placemark Label Mode", () => {
     );
```
Comment (vercasson) [inline cypress/e2e/placemark-label-mode.cy.ts:11]: all eslint

File: src/analytics.ts [hunk 6]
```diff
@@ -55,21 +55,25 @@ export async function sendAnalyticsCodeEvent(
     user_agent: window.navigator.userAgent, // User Agent
     z: Math.random().toString(36).substring(7), // Cache Buster (per google)
   };
- 
+
```
Comment (vercasson) [inline src/analytics.ts:58]: all eslint

**Review by amyhliao — APPROVED** [review]

**Comment by vercasson** [comment]
@tarakreft  I'm gonna merge this to get my other PR to pass. Lemme know if you see anything that's off

**Pull request merged**